postman.json

```
@@ -2752,26 +2908,22 @@
 							"formdata": []
 						},
 						"url": {
-							"raw": "{{baseUrl}}/media/{{videoIdentifier}}",
+							"raw": "{{domain}}/api/v1/media/{{videoIdentifier}}",
 							"host": [
-								"{{baseUrl}}"
+								"{{domain}}"
 							],
 							"path": [
+								"api",
+								"v1",
 								"media",
 								"{{videoIdentifier}}"
-							],
-							"query": [
-								{
-									"key": null,
-									"value": "",
-									"disabled": true
-								}
 							]
 						}
 					},
 					"response": []
 				}
 			],
+			"description": "The Transmorpher Media server processes videos in a queue. When the encoding into regular and stream formats have completed, the client is notified through a call. The client uses the public key URL to check if the notification is from the right source.",
```

```suggestion
			"description": "The Transmorpher Media server processes videos in a queue. When the encoding into regular and stream formats is completed, the client is notified through a signed call. The client has to use the public key URL to verify the server's signature.",
```

Maybe we should also mention the server sends this to the api url specified when creating the user and link to the `ClientNotification` enum for all available notifications.

Or link to the readme section where this is mentioned.